collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/AutoSendTest.kt

```diff
@@ -205,4 +210,42 @@ class AutoSendTest {
 
         mainMenuPage.assertNumberOfFinalizedForms(1)
     }
+
+    @Test
+    fun whenFormHasAutoSend_formsAreSentInOldestFirstOrder() {
+        val application = ApplicationProvider.getApplicationContext<Application>()
```

Comment:
For both this and the test for manual submissions, I think we could assert on the order without needing to dive into the `InstancesRepository`. Given that `testDependencies.server.submissions` is ordered, you could just assert on the contents - 0 should contain "31" and 1 should contain "32".

Reply:
Done in https://github.com/getodk/collect/pull/6356/commits/d27d6a9c23e65d7c6063b3f025b0de8508004a17